components/results_table/commons/results_table.lua

```diff
@@ -293,6 +258,47 @@ function BaseResultsTable:buildNonTeamOpponentConditions()
 	return opponentConditions
 end
 
+-- todo: adjust once #1802 is done
```

Comment:
Isn't this for non-team rather than team?

Reply:
if we use opponentname then we would need it for teams too, right?